charms/precise/clearwater-ellis/hooks/programmable-multiple-relation-changed

```diff
@@ -16,6 +16,9 @@ then
   grep -v ^RESOLV_CONF= /etc/default/dnsmasq > /tmp/dnsmasq.$$
   mv /tmp/dnsmasq.$$ /etc/default/dnsmasq
   echo RESOLV_CONF=/etc/dnsmasq.resolv.conf >> /etc/default/dnsmasq
+  # Disable caching of negative responses - this prevents failures if we start querying before the
+  # DNS charm has updated its records
+  sed -i "s/#no-negcache/no-negcache/g" /etc/dnsmasq.conf
```

Comment:
I left this too long and forgot what we agreed. Was this just to say that:
* this is Ellis-specific because it sends a ping on startup
* also it's because we don't have a way of waiting until the DNS server has acted on the request?